### ISSUE TITLE
missing reflect-metadata shim on my Windows machine

### ISSUE THREAD

**Issue by GaryB432** [preboot/angular2-webpack#66]
Getting `Uncaught reflect-metadata shim is required when using class decorators.`

**Comment by Foxandxss** [comment]
dup of #65

**Comment by GaryB432** [comment]
Hmmm didn't see a fix on #65.

I added this to `vendor.ts` before Angular2
```javascript
// Fixes for Uncaught reflect-metadata shim is required when using class decorators
import "zone.js/dist/zone";
import "reflect-metadata";
```
Thanks for the great repo!

**Comment by Foxandxss** [comment]
If there were a fix on #65 it would be closed and not open :P

That is a temporary workaround if you wish.

angular2-polyfills needs to be loaded before everything, included webpack. Doing an import like you did works, but any user using your app could be running outside a zone so they will get no change detection.

That is the reason I changed how this work. Problem is that in Windows the plugin is broken.
